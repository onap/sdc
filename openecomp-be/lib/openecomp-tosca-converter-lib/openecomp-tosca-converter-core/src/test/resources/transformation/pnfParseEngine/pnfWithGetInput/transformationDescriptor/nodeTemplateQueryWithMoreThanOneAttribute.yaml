- transformation-for: nodeTemplate
  query:
    type: tosca.nodes.nfv.PnfExtCp
    properties:
      anObjectAttribute:
        anyAttribute: anyAttributeValue
  conversions:
    - query:
        type:
      to-name: type
      to-value:
        strategy: replace
        from: tosca.nodes.nfv.PnfExtCp
        to: org.openecomp.resource.cp.v2.extCP