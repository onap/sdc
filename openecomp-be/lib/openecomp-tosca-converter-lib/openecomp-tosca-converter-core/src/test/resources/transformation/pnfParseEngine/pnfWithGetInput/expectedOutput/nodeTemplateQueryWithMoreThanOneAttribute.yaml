topology_template:
  node_templates:
    pnfExtCp_3:
      type: org.openecomp.resource.cp.v2.extCP
    pnfExtCp_2:
      type: org.openecomp.resource.cp.v2.extCP
    pnfExtCp_1:
      type: org.openecomp.resource.cp.v2.extCP